app/styles/components/Header.scss

```diff
@@ -0,0 +1,32 @@
+// Header
+
+.Header {
+
```

Comment:
In this file, the spacing is a bit off. In your editor, try adding support for .editorconfig's. http://editorconfig.org/